stratos-client/scripts/check-exports.mjs

```
@@ -0,0 +1,111 @@
+const EXPECTED_MODULES = [
+  'attestation.js',
+  'discovery.js',
+  'lexicons.js',
+  'routing.js',
+  'scopes.js',
+  'types.js',
+  'verification.js',
+]
+
+if (!existsSync(entryPath)) {
+  console.error(
```

ESLint

[error] 'console' is not defined.

(no-undef)